data/fk_constraints_data.csv

```diff
@@ -0,0 +1,63 @@
+Table,Foreign Key,Referenced Table,Referenced Column,Unnamed: 4
```

Comment:
Beware Excel export features: here we have one extra column on the left (entitled "Unnamed: 4"). It can be dropped using a simple text editor.